Sources/PackageModel/UserToolchain.swift

```diff
@@ -453,6 +455,17 @@ public final class UserToolchain: Toolchain {
         self.swiftCompilerPath = swiftCompilers.compile
         self.architectures = destination.architectures
 
+        #if canImport(Darwin)
+        let toolchainPlistPath = self.swiftCompilerPath.parentDirectory.parentDirectory.parentDirectory.appending(component: "Info.plist")
+        if localFileSystem.exists(toolchainPlistPath), let toolchainPlist = try? NSDictionary(contentsOf: URL(fileURLWithPath: toolchainPlistPath.pathString), error: ()), let overrideBuildSettings = toolchainPlist["OverrideBuildSettings"] as? NSDictionary, let isSwiftDevelopmentToolchainStringValue = overrideBuildSettings["SWIFT_DEVELOPMENT_TOOLCHAIN"] as? String {
+            self.isSwiftDevelopmentToolchain = isSwiftDevelopmentToolchainStringValue == "YES"
+        } else {
+            self.isSwiftDevelopmentToolchain = false
+        }
+        #else
+        self.isSwiftDevelopmentToolchain = false
+        #endif
+
```

Comment:
formatting can help make this more readable

Reply:
IMO becomes completely unreadable through swiftformat, but I guess that's what we're committed to.